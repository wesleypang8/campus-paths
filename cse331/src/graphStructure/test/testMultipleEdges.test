#test multiple edges between nodes

#create graph
CreateGraph graph1

#add nodes
AddNode graph1 node1
AddNode graph1 node2

#add edge
AddEdge graph1 node1 node2 edge1
AddEdge graph1 node1 node2 edge2

#print children/ edges
ListChildren graph1 node1