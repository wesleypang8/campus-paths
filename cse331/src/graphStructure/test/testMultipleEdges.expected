#test multiple edges between nodes

#create graph
created graph graph1

#add nodes
added node node1 to graph1
added node node2 to graph1

#add edge
added edge edge1 from node1 to node2 in graph1
added edge edge2 from node1 to node2 in graph1

#print children/ edges
the children of node1 in graph1 are: node2(edge1) node2(edge2)
